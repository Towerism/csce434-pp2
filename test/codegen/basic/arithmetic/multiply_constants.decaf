int main() {
    Print(2 * 5);
}