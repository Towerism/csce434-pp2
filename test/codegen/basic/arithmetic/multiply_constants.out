Loaded: /usr/share/spim/exceptions.s
10